pensieve/cli.py

```diff
@@ -72,20 +73,7 @@ def convert(self, value, param, ctx):
 )
 
 
-@cli.command()
-@project_id_option
-@dataset_id_option
-@click.option(
-    "--date",
-    type=ClickDate(),
-    help="Date for which experiments should be analyzed",
-    metavar="YYYY-MM-DD",
-    required=True,
-)
-@experiment_slug_option
-@dry_run_option
-@secret_config_file_option
-def run(project_id, dataset_id, date, experiment_slug, dry_run, config_file):
+def run(project_id, dataset_id, date, experiment_slug, dry_run, config_file, rerun_config_changed):
```

Comment:
I don't see anything but the Click handler that calls this; can we fold them back together? Same for rerun_config_changed_cmd; I see the pattern you're working towards but functions we aren't testing with long lists of untyped positional parameters feel like an invitation to step on a rake. We could fix that some other ways :D (and should maybe consider type annotations and forcing pass-by-name for `rerun`) but "not having them" also seems good